src/python/lib/ensembl/compara/orthtools/orthofinder.py

```diff
@@ -0,0 +1,92 @@
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Methods for preparing input files and running OrthoFinder.
+
+This module provides::
+
+    :func:`prepare_input_orthofinder()` which copies input fasta files for OrthoFinder
+            into a specified working directory
+
+    :func:`run_orthofinder()` which runs OrthoFinder
+
+Typical usage example::
+
+    from ensembl.compara.orthtools import prepare_input_orthofinder
+    prepare_input_orthofinder('path/to/source_dir', 'path/to/target_dir')
+
+    from ensembl.compara.orthtools import run_orthofinder
+    run_orthofinder('path/to/input/fasta/files')
+
+    from ensembl.compara.orthtools import run_orthofinder
+    run_orthofinder('path/to/input/fasta/files', 16, 4)
```

Comment:
Since these two are linked, what about:
```suggestion
    >>> from ensembl.compara.orthtools import prepare_input_orthofinder, run_orthofinder
    >>> prepare_input_orthofinder('path/to/src_dir', 'path/to/dst_dir')
    >>> run_orthofinder('path/to/dst_dir')
    # Alternatively, you can pass additional arguments to OrthoFinder:
    >>> run_orthofinder('path/to/dst_dir', t=16, a=4)
```
I have found examples that use `>>>` when it is specific "code", and without it when they are command lines. Furthermore, this nomenclature allows you to show each line's output, e.g.:
```
>>> print('hello')
hello
>>> exit()
```
I will try to track these down and update them accordingly in my next Python code review.

Reply:
Yes, that could be combined. I wrote it the way I did to emphasise they could be used independently, picking just the one you need. We don't have any examples with `>>>` in the codebase and I find it nicer without it but I can see the point.

Reply:
You have a tiny one in `ensembl-compara/src/python/lib/ensembl/compara/utils/tools.py`, but in this case, since you are not producing any output, I will leave it to your preference.

Reply:
🤔  Although the functions don't `return` anything, outputs are produced and written on the disk (symlinks or OrthoFinder output).